src/components/DigitalCredentials/ScanCredentials.vue

```diff
@@ -1,17 +1,124 @@
+                size="160"
+                :value="appleUrl"
+              />
+            </v-col>
+
+            <!-- Credential Type Indicators -->
+            <v-col cols="12" lg="9">
+              <!-- Business Credential Info -->
+              <template v-if="hasBusinessCred">
+                <v-row no-gutters>
+                  <v-progress-circular
+                    v-if="true"
+                    indeterminate
+                    color="primary"
+                    class="my-0"
+                    :size="25"
+                    :width="3" />
+                  <v-icon v-else color="success">mdi-check</v-icon>
+                  <h4 class="pl-3">Business Digital Credential</h4>
+                </v-row>
+                <v-row no-gutters>
+                  <p class="pt-2 pl-9">Successfully saved to your digital wallet app.</p>
+                </v-row>
+              </template>
+
+              <!-- Business Relationship Credential Info -->
+              <template v-if="hasBusinessRelationshipCred">
+                <v-row no-gutters class="mt-8">
+                  <v-progress-circular
+                    v-if="true"
+                    indeterminate
+                    color="primary"
+                    class="my-0"
+                    :size="25"
+                    :width="3" />
+                  <v-icon v-else color="success">mdi-check</v-icon>
+                  <h4 class="pl-3">Business Relationship Digital Credential</h4>
+                </v-row>
+                <v-row no-gutters>
+                  <p class="pt-2 pl-9">Successfully saved to your digital wallet app.</p>
+                </v-row>
+              </template>
+
+            </v-col>
+          </v-row>
+        </v-card-text>
+      </v-card>
+    </article>
+
+    <!-- Navigate back section when no selections made -->
+    <article v-else id="no-credentials-msg" @click="back()">
+      <v-card flat rounded class="pa-3">
+        <v-card-text>
+          <v-row no-gutters>
+            <v-col cols="12">
+              <span class="font-16">
+                <v-icon class="pr-1" color="primary">mdi-arrow-left</v-icon>
+                Please select a digital credential you wish to generate in the previous step.
+              </span>
+            </v-col>
+          </v-row>
+        </v-card-text>
+      </v-card>
+    </article>
+
+  </section>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
+import { Component, Emit, Prop, Vue } from 'vue-property-decorator'
+import QrcodeVue from 'qrcode.vue'
+
+@Component({
+  components: {
+    QrcodeVue
+  }
+})
+export default class ScanCredentials extends Vue {
+  @Prop({ default: false })
+  readonly hasBusinessCred: boolean
+
+  @Prop({ default: false })
+  readonly hasBusinessRelationshipCred: boolean
 
-@Component({})
-export default class ScanCredentials extends Vue {}
+  readonly appleUrl = 'https://apps.apple.com/us/app/trinsic-wallet/id1475160728'
```

Comment:
this should be invitationUrl

Reply:
For sure, just plopped it in as a placeholder until the UI implements the api integration :)
Can update property name though now if you prefer!

Reply:
Updated 👍